docs/sources/reference/commandline/cli.md

```diff
@@ -482,10 +482,13 @@ simultaneously, screen sharing style, or quickly view the progress of your
 daemonized process.
 
 You can detach from the container (and leave it running) with `CTRL-p CTRL-q`
-(for a quiet exit) or `CTRL-c` which will send a `SIGKILL` to the container.
+(for a quiet exit) or `CTRL-c` which will send a `SIGINT` to the container.
 When you are attached to a container, and exit its main process, the process's
 exit code will be returned to the client.
 
+(Note that a process running as PID 1 inside a container will not
+automatically terminate on receipt of SIGINT or SIGTERM)
```

Comment:
If this is a "note", then is should probably be formatted as such (see the [style guide](https://github.com/docker/docker/blob/master/docs/sources/project/doc-style.md#notes)). **But** I'd wait with changing until it's confirmed that everything is *technically* correct. Perhaps the doc maintainers have another way to formulate it so that no "note" is required.